### PR TITLE
Update/account selector

### LINKED ISSUE #298: Update Switch Accounts page
Per Eva's feedback, update the `Switch Account`. 

# To Do 
1. - [ ] Update the design to match the updated design of the rest of the app.
1. - [ ] Instead of displaying existing accounts, update the interface to make it appear that they have to "Create" accounts, instead.
1. - [ ] Disable interface-style "account deletion" if account balance > 0.0FIL (obviously the account isn't actually deleted, only hidden from selection in the interface).
1. - [ ] Hide pagination unless > N accounts are created.
1. - [ ] Set non-current accounts to `opacity:0.5;`

# Initial design exploration:

First time visiting the screen:

image

After creating one account (two accounts total):

image

### PR DESCRIPTION
**Pull request by as-dr**
Closes #298 

Updates design:

image

# To Do 

1. - [ ] Connect `ButtonCard` (Create account) button to display a new account.
1. - [ ] Create "Delete" (i.e. hide) account button - where does it go?

### CODE REVIEW

**Comment by Schwartz10** [comment]
New copy

image

Loading

image

Default state (notice the next account index will be the glyph)
image


Error - one thing i like about this, coincidentally, is that the error card actually stands up higher a bit than the others. It draws a nice attention to it. We could adjust this if we wanted to

image

File: components/Shared/AccountCard/Error.jsx [hunk 1]
```diff
@@ -24,7 +24,6 @@ const AccountError = ({ errorMsg, onTryAgain }) => (
       <Text>Error</Text>
     </Box>
     <Box>
-      <Title>Ledger Device Problem</Title>
```
Comment (Schwartz10) [inline components/Shared/AccountCard/Error.jsx:27]: @as-dr notice this change - idk if this fucks things up for you but not all errors are going to be Ledger related, so I just took this line out. If we want better ledger device errors (now we should recheck) we can just enhance the messages that are passed to this component. That should be done here https://github.com/openworklabs/filecoin-web-wallet/blob/primary/utils/ledger/reportLedgerConfigError.js#L1 so it should be easy to change (maybe we just add Ledger before device in all those messages)

**Comment by Schwartz10** [comment]
@as-dr this is ready for your review. I'm gonna add more tests later once I add some frameworks for testing the UI. Figured it wasn't fit for this PR, so just added snapshots for now. 

Test it with a ledger, and test it with different ledger error states too

**Comment by as-dr** [comment]
image

Do we want to conditionally change "seed phrase" here, depending on how they've accessed Glif?

E.g. In this case, I logged in my Ledger so it would read ```Your Ledger device creates hundreds of individual "accounts".``` or ```The single seed phrase within your Ledger device creates hundreds of individual "accounts".```

The goal being to try and minimize ambiguity / confusion.

File: components/Shared/AccountCard/Error.jsx (hunk 1 above)
Comment (as-dr) [reply components/Shared/AccountCard/Error.jsx:27]: This sounds reasonable.

Comment (as-dr) [reply components/Shared/AccountCard/Error.jsx:27]: @Schwartz10 

A user unplugs their Ledger
Clicks "Create Account"
Gets an error.

If the Ledger stays unplugged &&

If they click on the Error Card's "Try Again" button they are presented with the option to create an account again.

If they click on the card - to try and create another account - but nothing happens.

Shouldn't we display the ErrorCard again?

**Comment by as-dr** [comment]
When a user clicks on (i.e. selects) a non-current account, the AccountSelector page should close automatically, taking them to the wallet homepage.

File: components/Shared/AccountCard/Error.jsx (hunk 1 above)
Comment (Schwartz10) [reply components/Shared/AccountCard/Error.jsx:27]: I can't replicate this @as-dr

**Pull request merged**